Modules/Devices/src/Devices.Application/Identities/Commands/CancelDeletionProcessDuringGracePeriod/CancelDeletionProcessDuringGracePeriodCommand.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs.Models;
+using MediatR;
+
+namespace Backbone.Modules.Devices.Application.Identities.Commands.CancelDeletionProcessDuringGracePeriod;
+
+public class CancelDeletionProcessDuringGracePeriodCommand : IRequest<CancelDeletionProcessDuringGracePeriodResponse>
+{
+    public CancelDeletionProcessDuringGracePeriodCommand(string id)
+    {
+        CancelProcessId = id;
+    }
+
+    public string CancelProcessId { get; set; }
```

Comment:
@HunorTotBagi Why did you call it `CancelProcessId` and not just `ProcessId`?

Reply:
Regarding the comment of coderabbit: in theory it is of course better to make the setter private. But as all the other commands and queries have public setters as well, you can leave it public.

Reply:
@tnotheis Nikola Vetnić suggested a name change for this property to be `DeletionProcessId` that I already applied before I saw this comment. If I change it to `ProcessId` as you suggested, there will be a discrepancy with `ApproveDeletionProcessCommand` where this property is already called `DeletionProcessId`. I think it would be nice that this property is called the same in both commands. With what naming should we proceed?

Reply:
I like Nikola's approach better.